packages/x-charts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts",
-  "version": "6.0.0-alpha.0",
+  "version": "6.0.0-alpha.1",
```

Comment:
IMHO, we can skip updating this package version as long as we are not releasing 🤔 
@alexfauquette

Reply:
We can add a `--no-private` flag to `release:version` script to skip version bump for private packages:

```diff
-"release:version": "lerna version --exact --no-changelog --no-push --no-git-tag-version",
+"release:version": "lerna version --exact --no-changelog --no-push --no-git-tag-version --no-private",
```